docs/terms.md

```diff
@@ -0,0 +1,142 @@
+## Suite
+
+A suite of tests.
+A single suite has a directory structure, and many _test spec files_.
+Each test spec file is identified by its path within the suite.
+
+**Example:** `src/suites/*/`
+
+### README
+
+**Example:** `src/suites/**/README.txt`
+
+Describes (in prose) the contents of a subdirectory in a suite.
```

Comment:
nit: Will they be surfaced somewhere in the runner UI?

Reply:
They are surfaced in the standalone runner (kind of poorly for now, but it's there). It was broken for a while but I fixed it recently.